lti_consumer/lti_1p3/consumer.py

```diff
@@ -877,6 +895,15 @@ def generate_launch_request(
 
         return super().generate_launch_request(preflight_response)
 
+    def _check_if_scope_is_valid(self, scope):
+        """
+        Override of Lti1p3Consumer's _check_if_scope_is_valid
+        that accounts for the ACS scope for proctoring.
+        """
+        if scope in (LTI_1P3_ACCESS_TOKEN_SCOPES, LTI_1P3_ACS_SCOPE):
+            return True
+        return False
```

Comment:
I think you should be able to cover this missing case with a test for a token with a non-valid scope.